build.install4j

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="6.1.5" transformSequenceNumber="5">
+<install4j version="6.1.6" transformSequenceNumber="5">
```

Comment:
Until we complete the upgrade to install4j 6.1.6, it would probably be best to generate this file using 6.1.5.